### PR TITLE
Add some actions permissions

### PR DESCRIPTION
**Pull request by petamoriken**
## [ridedott/merge-me-action](https://github.com/ridedott/merge-me-action)

This Action approves and attempts to merge Pull Requests when triggered.

## [shogo82148/actions-check-permissions](https://github.com/shogo82148/actions-check-permissions)

A GitHub Action for checking the `contents` permissions of GITHUB_TOKEN.

FYI: https://shogo82148.github.io/blog/2021/03/17/actions-check-permissions/

## [saucelabs/sauce-connect-action](https://github.com/saucelabs/sauce-connect-action)

A GitHub action to launch Sauce Connect Proxy. ~~This action requires an access token, so it requires `contents: write` permissions.~~

FYI: https://github.com/saucelabs/sauce-connect-action/issues/24

## [codecov/codecov-action](https://github.com/codecov/codecov-action)

Easily upload coverage reports to Codecov from GitHub Actions. This action requires an access token for **private repositories**. ~~If you add to the logic whether or not an access token is passed to this action, it can detect permissions more flexibly.~~

### CODE REVIEW

File: actions.yml [hunk 1]
```diff
@@ -130,3 +130,14 @@ github/codeql-action/analyze:
     contents: read
     pull-requests: read
     security-events: write
+ridedott/merge-me-action:
```
Comment (azu) [inline actions.yml:133]: https://github.com/ridedott/merge-me-action/blob/7337d121e5e14af8ff1485eaab95caeeea2cf24d/src/common/getPullRequestInformation.ts#L95-L121
Probably, need to  `pull-requests: read` addtionarly.

File: actions.yml [hunk 2]
```diff
@@ -130,3 +130,14 @@ github/codeql-action/analyze:
...
+  permissions:
+    contents: write
+shogo82148/actions-check-permissions:
```
Comment (azu) [inline actions.yml:136]: OK: Yes, we can just ignore.
https://shogo82148.github.io/blog/2021/03/17/actions-check-permissions/

File: actions.yml [hunk 3]
```diff
@@ -130,3 +130,14 @@ github/codeql-action/analyze:
...
+# https://github.com/saucelabs/sauce-connect-action/issues/24
+saucelabs/sauce-connect-action:
```
Comment (azu) [inline actions.yml:138]: OK: https://github.com/saucelabs/sauce-connect-action/search?q=GITHUB_TOKEN
No use github token.

File: actions.yml [hunk 4]
```diff
@@ -130,3 +130,14 @@ github/codeql-action/analyze:
...
+  permissions:
+    contents: write
```
Comment (azu) [inline actions.yml:140]: Is this `contents`  permissions required?
In other words,  Does saucelabs/sauce-connect-action push to GitHub repository?

File: actions.yml [hunk 5]
```diff
@@ -130,3 +130,14 @@ github/codeql-action/analyze:
...
+codecov/codecov-action:
+  permissions:
+    contents: write # not required for public repos
```
Comment (azu) [inline actions.yml:143]: Is this means that private repo requires `write` permission?

**Review by azu — CHANGES_REQUESTED** [review]

File: actions.yml (hunk 5 above)
Comment (azu) [reply actions.yml:143]: Related: https://github.com/codecov/codecov-action/issues/300

File: actions.yml (hunk 4 above)
Comment (petamoriken) [reply actions.yml:140]: saucelabs/sauce-connect-action requires an access token, so probably we should use GitHub secrets. 
I think it requires `contents: write` permission, is that wrong?

Comment (petamoriken) [reply actions.yml:140]: For example, workflows triggered by `push` of dependabot have only read permission, so they cannot access secrets.

https://github.blog/changelog/2021-02-19-github-actions-workflows-triggered-by-dependabot-prs-will-run-with-read-only-permissions/

Comment (azu) [reply actions.yml:140]: https://github.com/saucelabs/sauce-connect-action#accesskey
This Actions requires SaurcesLab's accesskey, not `secrets.GITHUB_TOKEN`.
We do not require any `permissiosn` when use `${ secrets.FOO_SECRET_KEY }`.

This `permissions` value is for `secrets.GITHUB_TOKEN`.
https://github.blog/changelog/2021-04-20-github-actions-control-permissions-for-github_token/

I think that this actions does not require any `permissions` for `secrets.GITHUB_TOKEN`.

Comment (petamoriken) [reply actions.yml:141]: Sorry, I was wrong.
I experimented and found that access to secrets depends on whether it was forked or not, not on `contents: write` permissions.

File: actions.yml (hunk 5 above)
Comment (petamoriken) [reply actions.yml]: I removed contents permission.

**Review by azu — APPROVED** [review]

**Comment by azu** [comment]
Thanks!
https://github.com/pkgdeps/update-github-actions-permissions/releases/tag/v1.7.0 is released

**Pull request merged**